model_trainer.py

```
@@ -0,0 +1,42 @@
+
+# Train model
+model = LogisticRegression(max_iter=1000)
+model.fit(X_train, y_train)
+
+# Evaluate
+y_pred = model.predict(X_test)
+accuracy = accuracy_score(y_test, y_pred)
+print(f"✅ Model trained with {accuracy:.2f} accuracy")
+
+# Save the model
+pickle.dump(model, open("loan_model.pkl", "wb"))
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)